tests/integration/components/paper-dialog-test.js

```diff
@@ -252,3 +252,13 @@ test('opening gives focus', function(assert) {
   });
 
 });
+
+test('can specify dialog container classes', function(assert) {
+  this.render(hbs`
+    <div id="paper-wormhole"></div>
+    {{paper-dialog dialogContainerClasses='flex-50 my-dialog-container'}}
```

Comment:
For consistency project-wide, please use double quotes in everything template related.